edgelet/iotedge/Cargo.toml

```
@@ -32,7 +32,7 @@ sysinfo = "0.25"
 tabwriter = "1"
 termcolor = "1"
 thiserror = "1"
-tokio = { version = "1", features = ["macros", "process"] }
+tokio = { version = "1", features = ["macros", "parking_lot", "process"] }
```

```suggestion
tokio = { version = "1", features = ["macros", "parking_lot", "process", "rt", "sync"] }
```
`parking_lot` does not expose `tokio::sync`, and this crate uses `tokio::sync`. `rt` is required for `tokio::main`. For consistency with `aziot-edged`, we may want to use `rt-multi-thread`.